compilers/concrete-compiler/compiler/include/concretelang/ClientLib/ClientLib.h

```diff
@@ -33,6 +34,14 @@ using concretelang::values::Value;
 namespace concretelang {
 namespace clientlib {
 
+Result<TransportValue>
+importTfhersFheUint8(llvm::ArrayRef<uint8_t> serializedFheUint8,
```

Comment:
naming serialize/deserialize could be a better naming?

Reply:
Not using those was motivated by the fact that it's not actually a ser/deserialization, but a conversion first, followed by a serialization